lib/cell/option.rb

```diff
@@ -0,0 +1,35 @@
+require "trailblazer/option"
+require "uber/callable"
+
+# DISCUSS: Should we move this to trb-option instead ?
+# This is identical to `Representable::Option`.
+module Cell
+  # Extend `Trailblazer::Option` to support static values as callables too.
+  class Option < ::Trailblazer::Option
+    def self.callable?(value)
+      [Proc, Symbol, Uber::Callable].any?{ |kind| value.is_a?(kind) }
```

Comment:
I'm not sure about this change. We quite frequently pass symbols as cell options, is this going to to try and evaluate them as a method name?

Reply:
@samstickland This behaviour is same as the previous `Declatative::Option`. The difference with `Trailblazer::Option` in the API only, and it's compatible with Ruby-3.

Reply:
@yogeshjain999 We do have static values covered in the `cells` suite, I guess? :sweat_smile:

Reply:
Oh really ? I thought those are covered in declarative-option directly.